docs/docs/dev/documentation/dpp-specification.md

```diff
@@ -0,0 +1,47 @@
+# Data plane proxy specification
+
+The [`Dataplane`](#dataplane-entity) entity includes the networking and naming configuration that a data-plane proxy (`kuma-dp`) must have attempting to connect to the control-plane (`kuma-cp`).
+
+This specification is useful mostly in Universal and for troubleshooting as on Kubernetes Kuma-cp will generate it for you.
+
+The `Dataplane` entity includes a few sections:
+
+* `type`: must be `Dataplane`.
+* `mesh`: the `Mesh` name we want to associate the data-plane with.
+* `name`: this is the name of the data-plane instance, and it must be **unique** for any given `Mesh`. We might have multiple instances of a Service, and therefore multiple instances of the sidecar data-plane proxy. Each one of those sidecar proxy instances must have a unique `name`.
+* `networking`: this is the meaty part of the configuration. It determines the behavior of the data-plane on incoming (`inbound`) and outgoing (`outbound`) requests.
+    * `address` IP address or domain name at which this dataplane will be accessible to other data plane proxies. Domain name will be resolved to IP in the control plane.
```

Comment:
Might want to fix https://github.com/kumahq/kuma-website/issues/613 at the same time

Reply:
I deliberately avoid changing content in this PR because it won't be possible to review diff properly. This PR is only about moving and splitting pieces, let me do these content fixes in the follow up PR